dashboard/controller/activities/util/index.js

```diff
@@ -0,0 +1,48 @@
+// include libraries
+var request = require('request'),
+    common = require('../helper/common');
+
+function getActivity(req, aid, callback) {
```

Comment:
why does `getActivity` is inside utils?

Reply:
The functions defined in utils are being used only in launch and fakeLaunch functions and are private to activity controllers. So, to avoid writing redundant code, I put them inside utils and then imported them from there, so that it can be used in launch and fakeLaunch controllers. 
What are your suggestions?

> why does `getActivity` is inside utils?

Reply:
@imdhruvgupta: okay,  makes sense.
Could you also look at the `.DS_Store` file. I can still that in recent commit

Reply:
@tarunsinghal92 There were multiple .DS_Store files. I have removed them all now.